code/playerman/playercontrol.cpp

```diff
@@ -1600,11 +1600,6 @@ int player_inspect_cap_subsys_cargo(float frametime, char *outstr)
 		return 0;
 	}
 
-	// don't scan cargo on turrets, radar, etc.  only the majors: fighterbay, sensor, engines, weapons, nav, comm
-	if (!valid_cap_subsys_cargo_list(subsys->system_info->subobj_name)) {
```

Comment:
Some time ago, valid_cap_subsys_cargo_list was changed to always return 1. As a result, this block will never be executed; I have removed that function and all calls to it in this PR.